<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>360</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>520</width>
    <height>360</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>520</width>
    <height>360</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#widget{background-image: url(:/img.png);}</string>
  </property>
  <widget class="DynamicLineEdit" name="lineEdit_1">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>IP</string>
   </property>
  </widget>
  <widget class="AnimatedButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>240</y>
     <width>200</width>
     <height>42</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>42</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>42</height>
    </size>
   </property>
   <property name="text">
    <string>connect！</string>
   </property>
  </widget>
  <widget class="DynamicLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>159</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>端口</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DynamicLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">dynamiclineedit.h</header>
  </customwidget>
  <customwidget>
   <class>AnimatedButton</class>
   <extends>QPushButton</extends>
   <header location="global">animatedbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
